packages/components/src/Glimmer/Glimmer.tsx

```diff
@@ -46,15 +62,100 @@ export function Glimmer({
   size = "base",
   shape = "rectangle",
   timing = "base",
+  reverseTheme = false,
+  width,
 }: GlimmerProps) {
   const className = classnames(
     styles.glimmer,
     sizes[size],
     shapes[shape],
     timings[timing],
+    { [styles.reverseTheme]: reverseTheme },
+  );
+
+  return (
+    <div
+      aria-busy="true"
+      className={className}
+      data-testid={GLIMMER_TEST_ID}
+      style={{ width }}
+    />
+  );
+}
+
+interface GlimmerHeaderProps extends Omit<GlimmerProps, "shape" | "size"> {
+  /**
+   * Adjust the size of the `Glimmer.Header`.
+   *
+   * @default 3
+   */
+  readonly level?: 1 | 2 | 3 | 4 | 5;
+}
+
+Glimmer.Header = function GlimmerHeader({
```

Comment:
I am blind... where are we exporting `Glimmer.Header` and some of the other new components?? I wonder if these new `Glimmer.*` components would be easier to find in their own files.

Reply:
Oh! OHH! I get it now. lol
I do still think there would be value in having GlimmerHeader come from its own file... I think that is possible, right?

Reply:
It's not a deal breaker... I'll approve this and if you feel like changing it you can just ping me for re-review.

Reply:
lemme think about it. i'll leave it for now tho since it won't change how you use it